From: Carlos Alberto Lopez Perez <clopez@igalia.com>
Date: Tue, 8 Nov 2011 20:11:40 +0000
Subject: rename-to-wlan0

Origin: https://bugs.debian.org/648169

renames the interface ethX that the driver automatically
generates to another called wlanX like all the wireless drivers.
---
 src/wl/sys/wl_linux.c | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/wl/sys/wl_linux.c b/src/wl/sys/wl_linux.c
index 0d05100..947cef3 100644
--- a/src/wl/sys/wl_linux.c
+++ b/src/wl/sys/wl_linux.c
@@ -217,7 +217,7 @@ module_param(nompc, int, 0);
 #define to_str(s) #s
 #define quote_str(s) to_str(s)
 
-#define BRCM_WLAN_IFNAME eth%d
+#define BRCM_WLAN_IFNAME wlan%d
 
 static char intf_name[IFNAMSIZ] = quote_str(BRCM_WLAN_IFNAME);
 
